tests/unit/RestorationTypeSummary.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { shallowWrapperFactory } from '../jest-wrapper-factory'
+import RestorationTypeSummary from '@/components/Restoration/RestorationTypeSummary.vue'
+
+describe(`Initialize Restoration Type Component`, () => {
+  it('renders the component properly', () => {
+    const wrapper = shallowWrapperFactory(
+        RestorationTypeSummary
+    )
+
+    // verify the component is rendered
+    expect(wrapper.find('#restoration-type-summary').exists()).toBe(true)
+  })
+})
```

Comment:
This test suite should also verify that store properties display correctly, plus any other things that this component does.